projects/core/src/lib/colors/color-palette-editor/color-palette-editor.component.html

```diff
@@ -0,0 +1,12 @@
+<button (click)="debugClick()">Print Debuginfo</button>
```

Comment:
```suggestion
```

Reply:
Removed in [9a4a376](https://github.com/geo-engine/geoengine-ui/pull/598/commits/9a4a3768c86a47372dfbd81a1f27a822ae2bf847)